src/main/kotlin/frc/robot/subsystems/ExampleSubsystem.kt

```diff
@@ -4,22 +4,23 @@
 package frc.robot.subsystems
 
 import edu.wpi.first.math.system.plant.DCMotor
+import edu.wpi.first.math.system.plant.LinearSystemId
 import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax
 import edu.wpi.first.wpilibj.simulation.DCMotorSim
 import edu.wpi.first.wpilibj2.command.Command
 import edu.wpi.first.wpilibj2.command.SubsystemBase
 
-/** Creates a new ExampleSubsystem.  */
+/** Creates a new ExampleSubsystem. */
 class ExampleSubsystem(
     motorID: Int,
     /**
-     * An example of a property that is also a constructor argument.
-     * The value of inputID depends on the value passed in
-     * when the ExampleSubsystem is created.
+     * An example of a property that is also a constructor argument. The value of inputID depends on
+     * the value passed in when the ExampleSubsystem is created.
      */
     private val toPrint: Int,
-): SubsystemBase() {
+) : SubsystemBase() {
```

Comment:
Ew. Spotless doesn't remove this?

Reply:
Heh I think spotless changed it actually